tests/pytests/test_search_params.py

```diff
@@ -395,12 +453,38 @@ def test_numeric_range(env):
     env.expect('FT.SEARCH', 'idx', '@numval:[(-inf]').error()
     env.expect('FT.SEARCH', 'idx', '@numval:[($param]',
                'PARAMS', 2, 'param', 100).error()
-    env.expect('FT.SEARCH', 'idx', '@numval:[-$param]',
-               'PARAMS', 2, 'param', 100).error()
     env.expect('FT.SEARCH', 'idx', '@numval:[1  (($param]',
                'PARAMS', 2, 'param', 100).error()
     env.expect('FT.SEARCH', 'idx', '@numval:[(($param  1000]',
                'PARAMS', 2, 'param', 100).error()
+	# invalid syntax - multiple signs before parameters are not allowed
+    env.expect('FT.SEARCH', 'idx', '@n:[+-$n -+$n]', 'PARAMS', 2, 'n', 1).error()
+    env.expect('FT.SEARCH', 'idx', '@n:[++$n --$m]', 'PARAMS', 2, 'n', 1).error()
```

Comment:
Is there any special error that is raised here (I suspect no)?
You can add the `error().contains('...`)` function to verify the error thrown.

Reply:
No. there is not an special error message, only `Syntax error`